samples/corda/corda-simple-application/clients/src/main/kotlin/com/cordaSimpleApplication/client/HouseTokenTransferManager.kt

```diff
@@ -0,0 +1,542 @@
+/*
+ * Copyright IBM Corp. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.cordaSimpleApplication.client
+
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.core.requireObject
+import com.github.ajalt.clikt.parameters.arguments.argument
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.options.flag
+import com.github.ajalt.clikt.parameters.options.default
+import java.lang.Exception
+import java.io.File
+import kotlinx.coroutines.runBlocking
+import kotlin.system.exitProcess
+import net.corda.core.messaging.startFlow
+import org.json.JSONObject
+import java.util.Base64
+import java.util.Calendar
+import com.weaver.corda.app.interop.states.AssetClaimStatusState
+import com.weaver.corda.app.interop.states.AssetPledgeState
+import com.weaver.corda.app.interop.flows.GetAssetPledgeStatus
+
+import com.weaver.corda.sdk.AssetManager
+import com.cordaSimpleApplication.contract.AssetContract
+
+import net.corda.samples.tokenizedhouse.flows.GetAssetClaimStatusByPledgeId
+import net.corda.samples.tokenizedhouse.flows.GetAssetPledgeStatusByPledgeId
+import net.corda.samples.tokenizedhouse.flows.GetOurCertificateBase64
+import net.corda.samples.tokenizedhouse.flows.GetOurIdentity
+import net.corda.samples.tokenizedhouse.flows.RetrieveStateAndRef
+import net.corda.samples.tokenizedhouse.flows.GetIssuedTokenType
+import net.corda.samples.tokenizedhouse.states.FungibleHouseTokenJson
+import com.google.gson.Gson
+import com.google.gson.GsonBuilder
+
+import com.r3.corda.lib.tokens.contracts.commands.RedeemTokenCommand
+import com.r3.corda.lib.tokens.contracts.commands.IssueTokenCommand
+import com.r3.corda.lib.tokens.contracts.FungibleTokenContract
+import com.weaver.corda.app.interop.flows.RetrieveNetworkId
+import net.corda.core.identity.CordaX500Name
+import net.corda.core.identity.Party
+
+class HouseTokenTransferCommand : CliktCommand(name = "transfer", help ="Manages house-token asset transfer") {
```

Comment:
I feel for house token we can avoid this, as we already have one group for all fungible house tokens commands. As the command names in this file aren't same as any existing house token command. Feels unncessary to me, but its your decision whether to retain or remove it.

Reply:
As discussed keeping the `house-token` commands separate as it provides clarity.